src/button/styles/button.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
+	border-radius: 5px;
+	color: var(--button-color);
+	cursor: pointer;
+	font: var(--base-font);
+	font-weight: bold;
+	display: inline-block;
+	padding: 0.75em 1.25em;
+	transition: all 0.3s ease;
+}
+
+.button:hover,
+.button:focus,
+.button[aria-pressed="true"] {
+	background-color: var(--button-hover-bg);
+	border-color: var(--button-hover-bg);
+	outline: none;
+}
+
+.button[disabled],
+.button[disabled]:hover {
+	background-color: var(--button-disabled-bg);
+	border-color: var(--button-disabled-bg);
+	cursor: default;
+}
+
+.button[data-dojo-icon]:before {
```

Comment:
I'm happy to switch the attribute selector for a class. How would you pass the icon content without using a data-attr, though? Or would you prefer creating a child element just for the icon? It doesn't make a difference a11y-wise, but I've always preferred pseudo-elements for stuff like this.

For the font-weight, variant, etc. I added the same styles I use for every icon font I've ever come across. I have a hard time imagining an icon font that uses `font-weight: bold`, for instance :). I could definitely move the `font-size` to a variable, though.